funnel/templates/update_logo_modal.html.jinja2

```diff
@@ -1,6 +1,8 @@
 {% from "macros.html.jinja2" import faicon %}
 {% from "forms.html.jinja2" import renderfield %}
 
+<link rel="stylesheet" type="text/css" href="{{ built_asset('css/imgee-modal.scss') }}" />
```

Comment:
Shouldn't `<link>` tags be in the `<head>` section of HTML?

Reply:
This is a partial template opened in a modal via ajax